launch_ros/setup.py

```diff
@@ -12,6 +12,7 @@
         ('share/ament_index/resource_index/packages',
             ['resource/' + package_name]),
     ],
+    package_data={"launch_ros": ["py.typed"]},
```

Comment:
flake8 wants single instead of double quotes:

```suggestion
    package_data={'launch_ros': ['py.typed']},
```